katka/serializers.py

```diff
@@ -27,6 +29,8 @@
 from rest_framework import serializers
 from rest_framework.exceptions import PermissionDenied
 
+log = logging.getLogger("katka")
+
```

Comment:
not being used anymore? or is this a fix?

Reply:
thanks :) this can be removed idd